frontend/components.js

```diff
@@ -55,12 +69,21 @@ class UnlockClass extends Component {
                 size: '47',
                 value: currentWalletMnemonicOrSecret,
                 onInput: this.updateMnemonic,
+                autocomplete: 'nope',
```

Comment:
lol .. but I see that it works :)

Reply:
https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion